lib-ee/emqx_license/i18n/emqx_license_schema_i18n.conf

```diff
@@ -0,0 +1,81 @@
+emqx_license_schema {
+  license_root {
+    desc {
+      en: """
+      EMQX Enterprise license.
```

Comment:
HOCON does not support indentation at the moment.
So this will have to be reformatted to without indent.